### ISSUE TITLE
Problem while I try to run screen recorder

### ISSUE THREAD

**Issue by Rajagopalan-M** [kapoorlakshya/screen-recorder#99]
Last time I raised the same problem here

https://github.com/kapoorlakshya/screen-recorder/issues/85

But you have relaxed between 1 and 4 but right now, it's 4.1 so this problem shows up

Here is the error

```
C:\Ruby30\bin\ruby.exe C:/A/Rajagopalan/Run.rb
Input sheet =>"C:/A/Rajagopalan/InputFiles/Excel/NAPCIP/RND_PC/NAPCIPDelivery_Training_HO - E2E Flow V4.xlsx"
C:/Ruby30/lib/ruby/3.0.0/rubygems/specification.rb:2245:in `raise_if_conflicts': Unable to activate screen-recorder-1.5.0, because childprocess-4.1.0 conflicts with childprocess (>= 1.0, < 4.0) (Gem::ConflictError)
        from C:/Ruby30/lib/ruby/3.0.0/rubygems/specification.rb:1370:in `activate'
        from C:/Ruby30/lib/ruby/3.0.0/rubygems.rb:221:in `rescue in try_activate'
        from C:/Ruby30/lib/ruby/3.0.0/rubygems.rb:214:in `try_activate'
        from <internal:C:/Ruby30/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:153:in `rescue in require'
        from <internal:C:/Ruby30/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:149:in `require'
        from C:/A/Rajagopalan/Run.rb:39:in `<main>'
C:/Ruby30/lib/ruby/3.0.0/rubygems/specification.rb:2245:in `raise_if_conflicts': Unable to activate screen-recorder-1.5.0, because childprocess-4.1.0 conflicts with childprocess (>= 1.0, < 4.0) (Gem::ConflictError)
        from C:/Ruby30/lib/ruby/3.0.0/rubygems/specification.rb:1370:in `activate'
        from C:/Ruby30/lib/ruby/3.0.0/rubygems.rb:215:in `try_activate'
        from <internal:C:/Ruby30/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:153:in `rescue in require'
        from <internal:C:/Ruby30/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:149:in `require'
        from C:/A/Rajagopalan/Run.rb:39:in `<main>'
<internal:C:/Ruby30/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in `require': cannot load such file -- screen-recorder (LoadError)
        from <internal:C:/Ruby30/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from C:/A/Rajagopalan/Run.rb:39:in `<main>'

Process finished with exit code 1

```

**Comment by kapoorlakshya** [comment]
This should get fixed in the next release. Will relax `childprocess` requirement so v4 can be used.

**Comment by Rajagopalan-M** [comment]
> 
> 
> This should get fixed in the next release. Will relax `childprocess` requirement so v4 can be used.

Thanks.

**Comment by kapoorlakshya** [comment]
Just released v1.6.0 with support for `childprocess` v4.

**Comment by Rajagopalan-M** [comment]
@kapoorlakshya Thank you very much.
